static/app/views/replays/replayTable/tableCell.tsx

```diff
@@ -371,7 +376,7 @@ export function ReplayCell({
   );
 
   return (
-    <Item isWidget={isWidget} isReplayCell className={className}>
+    <Item isWidget={isWidget} isReplayCell className={className} onClick={handleClick}>
```

Comment:
i didn't realize the cell was clickable at first until i saw this code. it's probably because hovering over the cell doesn't indicate that it's clickable at all -- could we change the cursor for this case (e.g. if only `handleClick` is defined, change the cursor -- we don't want to change the cursor everywhere this cell appears)?

Reply:
Good idea